cinema/serializers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=100)
+    description = serializers.CharField()
```

Comment:
The `description` field in the `MovieSerializer` should allow `null` and `blank` values to match the `Movie` model's definition. Consider using `serializers.CharField(allow_blank=True, allow_null=True)`.